### PR TITLE
BETA: Register zlan.is-a.dev

### PR DESCRIPTION
**Pull request by zlance88**
Added `zlan.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/zlan.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "zlance88",
+        "email": "zlance88@outlook.com"
+    },
+    "record": {
+        "A": ["35.215.6.101"]
```
Comment (Stef-00012) [inline domains/zlan.json:7]: I cannot reach your website
"This site can’t be reached" (`DNS_PROBE_FINISHED_NXDOMAIN`)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/zlan.json (hunk 1 above)
Comment (zlance88) [reply domains/zlan.json:7]: The site was designed with another domain name "zleen.cf". But that domain DNS shutdown yesterday.

Comment (zlance88) [reply domains/zlan.json:7]: Shall I change it to "zlen.is-a.dev" now , to be approved?

Comment (Stef-00012) [reply domains/zlan.json:7]: > The site was designed with another domain name "zleen.cf". But that domain DNS shutdown yesterday.

I cannot reach `zleen.cf` too with same error

> Shall I change it to "zlen.is-a.dev" now , to be approved?

the `zlen` doesn't exist in the repo

Comment (zlance88) [reply domains/zlan.json:7]: sorry I'm confused. 
I'm trying to register a new domain, to be used for my website, which is not running yet. Is it required that site shall be up-and -running before approval?

Comment (Stef-00012) [reply domains/zlan.json:7]: > sorry I'm confused. I'm trying to register a new domain, to be used for my website, which is not running yet. Is it required that site shall be up-and -running before approval?

yes, if you see https://github.com/is-a-dev/register/blob/main/.github/PULL_REQUEST_TEMPLATE.md there's a check "You have completed your website." under the requirements

**Pull request closed without merging**